src/main/java/org/cryptomator/ui/fxapp/FxApplicationModule.java

```diff
@@ -70,4 +68,10 @@ static QuitComponent provideQuitComponent(QuitComponent.Builder builder) {
 		return builder.build();
 	}
 
+	@Provides
+	@FxApplicationScoped
+	static Dialogs provideDialogs(ResourceBundle resourceBundle) {
+		return new Dialogs(resourceBundle);
+	}
```

Comment:
This is redundant. Either provide a Dialogs Object via this provides method, _or_ use the `@Inject` annotation inside the Dialogs class.

You can test if with the `@Singleton` annotation you can inject the class without the need of a provides method.